java-gkehub/.OwlBot.yaml

```
@@ -28,13 +28,9 @@ deep-copy-regex:
   dest: "/owl-bot-staging/java-gkehub/v1beta/proto-google-cloud-gkehub-v1beta/src"
 - source: "/google/cloud/gkehub/v1beta1/.*-java/proto-google-.*/src/main/java/com/google/cloud/gkehub"
   dest: "/owl-bot-staging/java-gkehub/v1beta1/proto-google-cloud-gkehub-v1beta1/src/main/java/com/google/cloud/gkehub"
```

```suggestion
  - source: "/google/cloud/gkehub/(v.*)/.*-java/proto-google-.*/src"
    dest: "/owl-bot-staging/java-gkehub/$1/proto-google-cloud-alloydb-$1/src"
```
There's no `v1alpha2` in googleapis-gen. Could you change these two lines without specifying the version?